source/backend/api/Services/DocumentSyncService.cs

```diff
@@ -57,28 +57,40 @@ public ExternalBatchResult SyncMayanMetadataTypes(SyncModel model)
 
             // Add the metadata types not in mayan
             IList<Task<ExternalResult<MetadataType>>> createTasks = new List<Task<ExternalResult<MetadataType>>>();
+            IList<Task<ExternalResult<MetadataType>>> updateTasks = new List<Task<ExternalResult<MetadataType>>>();
             foreach (var metadataTypeLabel in model.MetadataTypes)
             {
-                if (metadata.Payload.Results.FirstOrDefault(x => x.Label == metadataTypeLabel) == null)
+                var matchingMetadataResult = metadata.Payload.Results.FirstOrDefault(x => x.Name == metadataTypeLabel.Name);
```

Comment:
because it matches by name, if the old metadata types names won't be the same and will end up being removed and new ones created. If it is in prod with metadata that might cause issues.  We need a one time sync that will convert the metadata on mayan to use the names before using the regular sync process